scenes/bear.tscn

```diff
@@ -0,0 +1,88 @@
+[gd_scene load_steps=6 format=3 uid="uid://cik241i7htjua"]
+
+[ext_resource type="Texture2D" uid="uid://b1ym7e0ndtpj3" path="res://characters/bear.png" id="1_8188a"]
+
+[sub_resource type="GDScript" id="GDScript_s40ds"]
+script/source = "extends Sprite2D
+
+@onready var bear_animation = $Bear/AnimationPlayer
+# Called when the node enters the scene tree for the first time.
+func _ready():
+	bear_animation.play(\"chase\")
+
+
+# Called every frame. 'delta' is the elapsed time since the previous frame.
+func _process(delta):
+	pass
```

Comment:
Did you add this code manually?

Reply:
Yeah, should the play be done another way?

Reply:
lol I actually have zero clue

Reply:
I'm guessing maybe I have to remove the script first in the inspector then delete the file? Cuz I had a bear.gd for a while then I realized I was doing the player animation in the other scene so I delete the script file, then went in the inspector and clicked clear but just now when I looked back there was this in the inspector attached the Bear node. but honestly idk really lol
![image](https://github.com/Mholliday6611/Stunt-Game/assets/22780615/a183c288-8816-413b-8e5c-43e8a9dcb9fb)